keyboards/pimoroni/keybow2040/info.json

```diff
@@ -0,0 +1,116 @@
+{
+    "manufacturer": "Pimoroni",
+    "keyboard_name": "Keybow 2040",
+    "url": "https://shop.pimoroni.com/products/keybow-2040",
+    "maintainer": "fanf2",
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "board": "GENERIC_RP_RP2040",
+    "features": {
+	"console": true
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": true,
+    },
+    "usb": {
+        "device_version": "1.0.0",
+        "vid": "0x07F8",
+        "pid": "0x4784",
```

Comment:
Unfortunately, that `0x07F8` value is too low — it's likely to be assigned to some company, but USB-IF does not make these assignments public, therefore any public databases of USB VIDs are definitely incomplete.  Currently the way to handle USB VID:PID values in QMK is mostly “pick some random number above `0x4000` for VID, check for collisions in any public databases and https://yanfali.github.io/qmk_usb_usage/, then just hope for the best”.

Reply:
Also fixed in round three